src/pages/Main/Main.js

```diff
@@ -1,9 +1,33 @@
 import React, { Component } from 'react';
+import './Main.scss';
+import MainVisual from './Mainvisual';
+import MainBestItem from './MainBestItem';
+
+class Main extends Component {
+  constructor() {
+    super();
+    this.state = {
+      bestList: [],
+    };
+  }
+
+  componentDidMount() {
+    // fetch('http://10.58.1.209:8000/products/1/1/1', {
+    fetch('http://localhost:3000/data/BestItemData.json')
```

Comment:
```suggestion
    fetch('/data/BestItemData.json')
```
localhost:3000 쓸 필요 없습니다!
바로 "/"로 시작하시면 지금 현재 돌아가고 있는 서버로 인식됩니다!

Reply:
수정완료했습니다~